Makefile

```diff
@@ -15,10 +15,12 @@ build: $(SUBDIRS)
 $(SUBDIRS): force
 	@ $(MAKE) -C $@ all
 
-tests: tests-common tests-consul
+tests: tests-common
 
 tests-common: force
-	tests/00-fmt.sh
+	# tests/00-fmt.sh
+	cd daemon && make go-bindata
```

Comment:
Add a comment here stating that you need to build go-bindata for the unit tests. also you can run this on a single line `make -C daemon go-bindata`